simapp/app.go

```diff
@@ -161,6 +161,9 @@ func NewSimApp(
 	app.subspaces[crisis.ModuleName] = app.ParamsKeeper.Subspace(crisis.DefaultParamspace)
 	app.subspaces[evidence.ModuleName] = app.ParamsKeeper.Subspace(evidence.DefaultParamspace)
 
+	// set the BaseApp's parameter store
+	bApp.SetParamStore(app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(BaseAppParamsKeyTable()))
```

Comment:
@marbar3778 @fedekunze @cwgoes would like to get feedback here.

The benefit is that consensus parameters can now be managed by x/params and thus be tweaked via governance. However, app devs will have to explicitly do this in their app constructor. Also, I'm not a fan of `BaseAppParamsKeyTable` living in simapp, but not sure of a better place to put it. Ideally, it'd be in `baseapp`, but we cannot rely on the `x/params` package.

Reply:
I think it's fine for it to be explicit.

Maybe we could put `BaseAppParamsKeyTable` in a subpackage of `baseapp` which does depend on `x/params`? Or is that still undesirable?

Reply:
yeah that could work too. Either in a subpackage of `baseapp/*` or in `x/params` itself.

Reply:
I'm going to leave this function in `simapp` for now.